spec/lib/submission_builder/ty2022/states/az/az_return_xml_spec.rb

```diff
@@ -141,5 +141,28 @@
         expect(builder_response.errors).not_to be_present
       end
     end
+
+    context "when 1099-rs are present" do
+      let(:intake) do
+        create(:state_file_az_intake,
+               raw_direct_file_data: StateFile::XmlReturnSampleService.new.read('az_richard_retirement_1099r'),
+               filing_status: "married_filing_jointly",
+               primary_received_pension: "yes",
+               primary_received_pension_amount: 2000.6,
```

Comment:
if it's possible to have `primary_received_pension`=yes but `primary_received_pension_amount` be nil, we should have a test that covers that

Reply:
it really shouldn't be possible though. Mike and I discussed while pairing, and if such a situation happens, we decided we would prefer a loud error to happen rather than silently handling it